src/components/TweetList/TweetList.tsx

```diff
@@ -1,9 +1,55 @@
 import * as React from 'react';
+import { observer, inject } from 'mobx-react';
 
 import './TweetList.scss';
 
-export const TweetList = () => (
-  <div className="content__tweets-list">
-    <h1>Tweets</h1>
-  </div>
-);
+import { Fetch } from '../../helpers/fetch';
+import { env } from '../../env/environment';
+
+import { ITweet } from '../../interfaces/tweet';
+import { IStores } from '../../interfaces/stores';
+import { TweetItem } from '../TweetItem';
+import { CreateTweet } from '../CreateTweet';
+
+export interface IAllTweetsProps {
+  stores?: IStores
+}
+
+@inject("stores")
+@observer
+export class TweetList extends React.Component<IAllTweetsProps, {}> {
+  constructor(props: IAllTweetsProps) {
+    super(props);
+  }
+
+  getAllPosts(): Promise<ITweet[] > {
+    const fetch = Fetch.request(env.securedRoutes + '/posts', {
```

Comment:
we could use
`return Fetch.request(******......`
instead returning const with it